src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-smart-list/osd-smart-list.component.html

```diff
@@ -0,0 +1,67 @@
+<ng-container *ngIf="!loading; else isLoading">
+  <ng-container *ngIf="incompatible; else isCompatible">
+    <cd-alert-panel type="warning" i18n>The data received has the JSON format version 2.x and is currently incompatible with the dashboard.</cd-alert-panel>
+  </ng-container>
+  <ng-template #isCompatible>
+    <tabset *ngFor="let device of data | keyvalue">
+      <tab [heading]="device.value.device + ' (' + device.value.identifier + ')'">
+        <ng-container *ngIf="device.value.error; else noError">
+          <cd-alert-panel type="warning">{{ device.value.userMessage }}</cd-alert-panel>
+        </ng-container>
+        <ng-template #noError>
+          <div class="btn-group mb-3" role="group" aria-label="Report Actions">
+            <button class="btn btn-primary"
+                    title="Copy to Clipboard"
+                    i18n-title
+                    cdToClipboard
+                    [data]="device.value.report">
+              <i class="fa fa-files-o" aria-hidden="true"></i>
+            </button>
+            <button class="btn btn-primary"
+                    title="Download"
+                    i18n-title
+                    [cdTextToDownload]="device.value.report"
+                    [filename]="device.value.reportFilename">
+              <i class="fa fa-download" aria-hidden="true"></i>
+            </button>
+          </div>
+
+          <ng-container *ngIf="device.value.smart.data.self_test.status.passed; else selfTestFailed">
+            <cd-alert-panel
+              [showIcon]="false"
+              size="slim"
+              type="info"
+              title="SMART overall-health self-assessment test result">
+              {{ device.value.smart.data.self_test.status.string }}
+            </cd-alert-panel>
+          </ng-container>
+          <ng-template #selfTestFailed>
+            <cd-alert-panel
+              [showIcon]="false"
+              size="slim"
+              type="warning"
+              title="SMART overall-health self-assessment test result">{{ device.value.smart.data.self_test.status.string }}</cd-alert-panel>
+          </ng-template>
+
+          <tabset>
+            <tab i18n-heading
+                 heading="Device Information">
+              <cd-table-key-value [renderObjects]="true"
+                                  [data]="device.value.info"></cd-table-key-value>
+            </tab>
+
+            <tab i18n-heading
+                 heading="S.M.A.R.T">
```

Comment:
Dotted? Unless we want to write C.R.U.S.H, R.B.D. or O.S.D., I'd change this to SMART for the sake of consistency.

Reply:
I've never seen R.B.D or O.S.D in the documentation of Ceph. S.M.A.R.T, on the other hand, is pretty common. But I'm not opposed to change it to SMART.

Reply:
You're right on that point, but I still find 'noisy' the `S.M.A.R.T.` form (2 x length) vs just `SMART`.